### PR TITLE
CLDR-14118 update likelySubtags and maximalLocales

### PR DESCRIPTION
**Pull request by srl295**
CLDR-14118

- defaultContent changed from os_GE to os_RU
- Exemplars: Renamed `izi` -> `eza` and `dud` -> `uth`
- Also exemplars: deleted `daf` - see [CLDR-14136]
- changed TestIdentity so it always runs on all files, not just exhaustive (12s to 27s)
- changed DeriveScripts to produce better error messages

[CLDR-14136]: https://unicode-org.atlassian.net/browse/CLDR-14136

### CODE REVIEW

File: common/supplemental/supplementalMetadata.xml [hunk 1]
```diff
@@ -1664,16 +1664,16 @@ For terms of use, see http://www.unicode.org/copyright.html
...
+			ka_GE kab_DZ kaj_NG kam_KE kcg_NG kde_TZ kea_CV ken_CM kgp_BR khq_ML ki_KE
+			kk_KZ kkj_CM kl_GL kln_KE km_KH kn_IN ko_KR kok_IN kpe_LR ks_Arab ks_Arab_IN
 			ks_Deva_IN ksb_TZ ksf_CM ksh_DE ku_TR kw_GB ky_KG
 			lag_TZ lb_LU lg_UG lkt_US ln_CD lo_LA lrc_IR lt_LT lu_CD luo_KE luy_KE lv_LV
 			mai_IN mas_KE mer_KE mfe_MU mg_MG mgh_MZ mgo_CM mi_NZ mk_MK ml_IN mn_MN
 			mn_Mong_CN mni_Beng mni_Beng_IN mni_Mtei_IN moh_CA mr_IN ms_Arab_MY ms_MY mt_MT
 			mua_CM mus_US my_MM myv_RU mzn_IR
 			naq_NA nb_NO nd_ZW nds_DE ne_NP nl_NL nmg_CM nn_NO nnh_CM nqo_GN nr_ZA nso_ZA
 			nus_SS nv_US ny_MW nyn_UG
-			oc_FR om_ET or_IN os_GE osa_US
```
Comment (macchiati) [inline common/supplemental/supplementalMetadata.xml:1676]: this is a geopolitical issue; Ossetia is still recognized by most countries as part of Georgia (https://en.wikipedia.org/wiki/South_Ossetia). It is probably coming from some of the changed population figures, so we should check those carefully, and maybe special-case them for now.

File: common/supplemental/likelySubtags.xml [hunk 2]
```diff
@@ -1926,8 +1934,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Orokolo; ?; ? } => { Orokolo; Latin; Unknown Region }-->
 		<likelySubtag from="oru" to="oru_Arab_ZZ"/>
 		<!--{ Ormuri; ?; ? } => { Ormuri; Arabic; Unknown Region }-->
-		<likelySubtag from="os" to="os_Cyrl_GE"/>
-		<!--{ Ossetic; ?; ? } => { Ossetic; Cyrillic; Georgia }-->
+		<likelySubtag from="os" to="os_Cyrl_RU"/>
```
Comment (macchiati) [inline common/supplemental/likelySubtags.xml:1937]: Geopolitical issue, see below.

**Comment by macchiati** [comment]
I think there are two approaches:
1. patch
a) modify supplementalData to fix os population figures

OR
2. fix generation
a1) Modify the update program for the supplementalData population figures 
... Ossetian speakers number about 614,350, with 451,000 speakers in the Russian Federation recorded in the 2010 census.[the rest being in GE]
a2) rerun the generation of the population figures.

then for both cases:
b) add a test that the following occur: 
<likelySubtag from="os" to="os_Cyrl_GE"/>
<defaultContent locales="... os_GE"
and the test message can say to patch the population figures to correct the problem, thus covering us in the future.
c) rerun the generation for this PR.

**Comment by macchiati** [comment]
In a way, #1 is simpler to do now, and less fragile in the future (the test message can remind us to check that the geopolitical situation hasn't changed.)

File: common/supplemental/likelySubtags.xml (hunk 2 above)
Comment (srl295) [reply common/supplemental/likelySubtags.xml:1937]: OK , would we ideally leave the likelySubtag  and default content as is?

**Comment by srl295** [comment]
>      Error: (TestBasic.java:745) Default content file not empty:	os_RU

This will be fixed when DC/likely is fixed.

**Comment by macchiati** [comment]
Right, we want to leave likelySubtag and default content as is, **for the
os lines**. (The other changes for likelySubtag and default content  are
ok).

Mark

**Comment by srl295** [comment]
(tracing) 

```
os      Ossetic OR      539,819 OR-Georgia [GE]-87,670  U-Asia [142]-87,670     U-United Nations [UN]-87,670    U-Western Asia [145]-87,670     U-World [001]-87,670
```

but in country_lang_pop_raw.txt:

```
Georgia	GE	"4,926,087"	100%	"39,850,000,000"	official_regional	Ossetic	os	"109,000"		Russia	RU	"142,122,776"	100%	"4,016,000,000,000"		Ossetic	os	"456,000"			http://www.gks.ru/free_doc/new_site/population/demo/per-itog/tab6.xls census data
```	

So, 73b6f99f0c8a283d6a0ea940a81c6a4047ba6302 had nary a mention of os[setic], but the population of `GE` dropped 20%.

**Comment by srl295** [comment]
OK. RU has 0.32% `os` speakers, so 435k (close to the figure above). GE has 2.2% so 88k.

If we change the GE % to 4.09% instead of 2.2%, then GE will have 163350 speakers. I think that's what you;'re proposing in `#1` above, i'll try that.

**Comment by srl295** [comment]
This is the "fix" patch, followed by the results of re-running `org.unicode.cldr.tool.GenerateMaximalLocales`

```diff
diff --git a/common/supplemental/supplementalData.xml b/common/supplemental/supplementalData.xml
index 93d9702e8d..ee819627c6 100644
--- a/common/supplemental/supplementalData.xml
+++ b/common/supplemental/supplementalData.xml
@@ -2967,7 +2967,7 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="ru" populationPercent="9" references="R1038"/>	<!--Russian-->
 			<languagePopulation type="hy" populationPercent="7" references="R1038"/>	<!--Armenian-->
 			<languagePopulation type="ab" populationPercent="2.2" officialStatus="official_regional"/>	<!--Abkhazian-->
-			<languagePopulation type="os" populationPercent="2.2" officialStatus="official_regional"/>	<!--Ossetic-->
+			<languagePopulation type="os" populationPercent="4.09" officialStatus="official_regional"/>	<!--Ossetic - per https://en.wikipedia.org/wiki/Ossetian_language -->
 			<languagePopulation type="ku" populationPercent="0.89"/>	<!--Kurdish-->
 		</territory>
 		<territory type="GF" gdp="1551000000" literacyPercent="83" population="199509">	<!--French Guiana-->
diff --git a/common/supplemental/likelySubtags.xml b/common/supplemental/likelySubtags.xml
index 43f837a499..18ce923315 100644
--- a/common/supplemental/likelySubtags.xml
+++ b/common/supplemental/likelySubtags.xml
@@ -1934,8 +1934,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Orokolo; ?; ? } => { Orokolo; Latin; Unknown Region }-->
 		<likelySubtag from="oru" to="oru_Arab_ZZ"/>
 		<!--{ Ormuri; ?; ? } => { Ormuri; Arabic; Unknown Region }-->
-		<likelySubtag from="os" to="os_Cyrl_RU"/>
-		<!--{ Ossetic; ?; ? } => { Ossetic; Cyrillic; Russia }-->
+		<likelySubtag from="os" to="os_Cyrl_GE"/>
+		<!--{ Ossetic; ?; ? } => { Ossetic; Cyrillic; Georgia }-->
 		<likelySubtag from="osa" to="osa_Osge_US"/>
 		<!--{ Osage; ?; ? } => { Osage; Osage; United States }-->
 		<likelySubtag from="ota" to="ota_Arab_ZZ"/>
@@ -3386,8 +3386,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ ?; Cyrillic; Albania } => { Macedonian; Cyrillic; Albania }-->
 		<likelySubtag from="und_Cyrl_BA" to="sr_Cyrl_BA"/>
 		<!--{ ?; Cyrillic; Bosnia & Herzegovina } => { Serbian; Cyrillic; Bosnia & Herzegovina }-->
-		<likelySubtag from="und_Cyrl_GE" to="ab_Cyrl_GE"/>
-		<!--{ ?; Cyrillic; Georgia } => { Abkhazian; Cyrillic; Georgia }-->
+		<likelySubtag from="und_Cyrl_GE" to="os_Cyrl_GE"/>
+		<!--{ ?; Cyrillic; Georgia } => { Ossetic; Cyrillic; Georgia }-->
 		<likelySubtag from="und_Cyrl_GR" to="mk_Cyrl_GR"/>
 		<!--{ ?; Cyrillic; Greece } => { Macedonian; Cyrillic; Greece }-->
 		<likelySubtag from="und_Cyrl_MD" to="uk_Cyrl_MD"/>
diff --git a/common/supplemental/supplementalMetadata.xml b/common/supplemental/supplementalMetadata.xml
index 04d82930de..8b40f5949c 100644
--- a/common/supplemental/supplementalMetadata.xml
+++ b/common/supplemental/supplementalMetadata.xml
@@ -1673,7 +1673,7 @@ For terms of use, see http://www.unicode.org/copyright.html
 			mua_CM mus_US my_MM myv_RU mzn_IR
 			naq_NA nb_NO nd_ZW nds_DE ne_NP nl_NL nmg_CM nn_NO nnh_CM nqo_GN nr_ZA nso_ZA
 			nus_SS nv_US ny_MW nyn_UG
-			oc_FR om_ET or_IN os_RU osa_US
+			oc_FR om_ET or_IN os_GE osa_US
 			pa_Arab_PK pa_Guru pa_Guru_IN pcm_NG pl_PL prg_001 ps_AF pt_BR
 			qu_PE quc_GT
 			rm_CH rn_BI ro_RO rof_TZ ru_RU rw_RW rwk_TZ

```

**Review by macchiati — DISMISSED** [review]
Looks great, thanks.

**Comment by srl295** [comment]
@macchiati adding a test… 
Although,  ` Error: (TestBasic.java:745) Default content file not empty:	os_RU` gives a sanity check.

**Comment by macchiati** [comment]
ok
Mark

**Comment by srl295** [comment]
@macchiati okay, *now* it should be ready for review and merge.

**Comment by srl295** [comment]
The previous fix that you reviewed (prior to the special test) passed CI :tada: https://github.com/unicode-org/cldr/runs/1038933265?check_suite_focus=true

**Pull request merged**